### PR TITLE
EPMFARMATS-12668

### PR DESCRIPTION
**Pull request by overdd**

### CODE REVIEW

File: EPMFARMATS-12668.js [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+//-2 - 2
+//���������� ����� ����: "+", "-", "*" ��� "/".
+//� ��������� ����� ���� ������� � ������, � ����� ��� ����� ������� ���������.
+//�������� ������� parse(expr), ������� ��������� ��������� � ���������� ������ �� ��� ���������:
+//������ �����.
+//��������.
+//������ �����.
+
+function parse(expr) {
+    let regExp = /(-?[0-9]+(?:\.[0-9]+)?)\s+([+-*\/])\s+(-?[0-9]+(?:\.[0-9]+)?)/
```
Comment (SiarheiRylach) [inline EPMFARMATS-12668.js:33]: have you checked how it works?

File: EPMFARMATS-12668.js [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+//P.S. ��� ������ ���� ����� 3 ��� 6 ����������������� ����. ��� ���� �������� � 4-�� ������� ���� #abcd �� ������ ��������� � ���������.
+
+let regExp = /(#([0-9a-f]{3}){1,2}\b)/gi;
+let str = "color: #3f3; background-color: #AA00ef; and: #abcd";
+alert(str.match(regExp)); // #3f3 #AA00ef
+
+//�������� ���������� ���������, ������� ���� ����� ���������� �����, ������� �������������, � ��������� ������ � �������������.
+//������ �������������:
+
+let regexp = /-?[0-9]+(\.[0-9]+)?/g;
```
Comment (SiarheiRylach) [inline EPMFARMATS-12668.js:15]: Doesn't work appropriate. You should exclude fractional part of number.

File: EPMFARMATS-12668.js (hunk 1 above)
Comment (overdd) [reply EPMFARMATS-12668.js:33]: Fixed

File: EPMFARMATS-12668.js (hunk 2 above)
Comment (overdd) [reply EPMFARMATS-12668.js:15]: Fixed

Comment (overdd) [reply EPMFARMATS-12668.js:15]: Execution result:

**Review by SiarheiRylach — APPROVED** [review]

**Pull request merged**